README.md

```diff
@@ -205,7 +205,7 @@ _It is all about the looks, apply some style._
 - [Black and Green](https://community.home-assistant.io/t/black-and-green-theme/28602?u=frenck) - A dark theme with pale green accents by GreenTurtwig.
 - [Vintage](https://community.home-assistant.io/t/vintage-theme/42806?u=frenck) - Give your frontend a vintage look with this theme by Anup Surendran.
 - [Carbon Green](https://community.home-assistant.io/t/share-your-themes/22018/95?u=frenck) - Light carbon theme with green accents by Reua.
-- [20 Great Themes](https://www.juanmtech.com/themes-in-home-assistant/) - 20 Great themes by JuanMTech (includes a guide).
+- [20 Great Themes](https://www.juanmtech.com/how-to-set-up-themes-in-home-assistant/) - 20 Great themes by JuanMTech (includes a guide).
```

Comment:
That link is now inactive / removed as well. A quick glance, it seems like there is no direct replacement for the original one. Removing it for that reason:

```suggestion
```